Website/src/rendering/src/components/Products/ProductDetails.tsx

```diff
@@ -21,7 +22,9 @@ const ProductDetails = (): JSX.Element => {
       return;
     }
 
-    trackPDPViewEvent(productSku);
+    if (isDiscoverEnabled) {
```

Comment:
I think we should add a `DiscoverService.trackPDPViewEvent(productSKU)` function that would check whether Discover is enabled before calling the Discover SDK function. We should do the same for other Discover SDK functions that we are using in components. Then we would not need to wrap every calls to the Discover SDK functions in components. We would just call the DiscoverService wrappers.